docs/components_page/helpers.py

```diff
@@ -15,13 +15,18 @@ def ExampleContainer(component):
     return html.Div(component, className="example-container")
 
 
-def load_source_with_app(app, source, component_name):
+def load_source_with_app(
+    app, source, component_name, globals_dict=None, locals_dict=None
```

Comment:
Are we ever using `locals_dict`?

Reply:
In fact, would it be better to inject additional local variables in `locals_dict`?

We could consider changing `load_source_with_app` to `load_source_with_locals` and folding passing `app` into the local variables.

This would then be called with:

```py
load_source_with_locals(
    tabs_active_source, 
    {'app': app, 'tab1_content': tab1_content, 'tab2_content': tab2_content}
)

Reply:
I like this, I'll have a go at making the relevant changes